.gitpod.Dockerfile

```
@@ -0,0 +1,7 @@
+FROM gitpod/workspace-full
+
+USER gitpod
+
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+
+RUN rustup default stable
```

Please note that there is an [official and dedicated rust _gitpod_ image](https://hub.docker.com/r/gitpod/workspace-rust). 

```suggestion
FROM gitpod/workspace-rust:2023-11-16-11-19-36

USER gitpod

```